reference/configuration/framework.rst

```diff
@@ -465,45 +953,211 @@ would be ``/images/logo.png?version=5``.
+
+                        <framework:resource>WebsiteBundle:Form</framework:resource>
+
+                    </framework:form>
+
+                </framework:templating>
+
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'templating' => array(
+                'form' => array(
+                    'resources' => array(
+                        'WebsiteBundle:Form'
+                    ),
+                ),
+            ),
+        ));
+
+.. note::
+
+    The default form templates from ``FrameworkBundle:Form`` will always be
+    included in the form resources.
+
+.. _reference-templating-base-urls:
+
+assets_base_urls
+................
+
+**type**: ``{ http: [], ssl: [] }``
+
+This option allows you to define base URLs to be used for assets referenced
+from ``http`` and ``ssl`` (``https``) pages. A string value may be provided in
+lieu of a single-element array. If multiple base URLs are provided, Symfony2
+will select one from the collection each time it generates an asset's path.
+
+For your convenience, ``assets_base_urls`` can be set directly with a string or
+array of strings, which will be automatically organized into collections of base
+URLs for ``http`` and ``https`` requests. If a URL starts with ``https://`` or
+is `protocol-relative`_ (i.e. starts with ``//``), it will be added to both
+collections. URLs starting with ``http://`` will only be added to the
+``http`` collection.
+
+.. _reference-templating-cache:
+
+cache
+.....
+
+**type**: ``string``
+
+The path to the cache directory for templates. When this is not set, caching is
+disabled. It becomes the service container parameter named
+``templating.loader.cache.path``.
+
+engines
 .......
 
-**default**: ``true`` in the ``dev`` and ``test`` environments
+**type**: ``string[]`` / ``string`` **required**
 
-The profiler can be disabled by setting this key to ``false``.
+The Templating Engine to use. This can either be a string (when only one engine
+is configured) or an array of engines. It becomes the service container
+parameter named ``templating.engines``.
 
-.. versionadded:: 2.3
-    The ``collect`` option was introduced in Symfony 2.3. Previously, when
-    ``profiler.enabled`` was ``false``, the profiler *was* actually enabled,
-    but the collectors were disabled. Now, the profiler and the collectors
-    can be controlled independently.
+At least one engine is required.
 
-collect
+loaders
 .......
 
-**default**: ``true``
+**type**: ``string[]``
 
-This option configures the way the profiler behaves when it is enabled. If set
-to ``true``, the profiler collects data for all requests. If you want to only
-collect information on-demand, you can set the ``collect`` flag to ``false``
-and activate the data collectors by hand::
+An array (or a string when configuring just one loader) of service ids for
+templating loaders.
```

Comment:
We should say what the purpose of a template loader is, and perhaps tell them the interface they should implement.